cmd/packaging.go

```diff
@@ -38,3 +39,14 @@ var initLinuxDebCmd = &cobra.Command{
 		packaging.InitLinuxDeb()
 	},
 }
+
+var initLinuxAppImageCmd = &cobra.Command{
+	Use:   "linux-appimage",
+	Short: "Create configuration files for AppImage packaging",
+	Run: func(cmd *cobra.Command, args []string) {
+		assertHoverInitialized()
+		packaging.DockerInstalled()
```

Comment:
This will print a warning, doesn't do anything else. Should we really do this here?

Reply:
Looks like I missed the `os.Exit(1)` there

Reply:
This was only meant as a warning, because I thought you should still be able to init packaging for a format without having docker installed.
Maybe you have other thoughts about this? Don't know whats the best way